src/components/pages/ImageClassification/sampleData/image_endpoints.md

```
@@ -0,0 +1,1455 @@
+          "growth_form": null,
+          "classifier": "b3af1f56-cc34-4fea-b9b3-40a6df99c485",
+          "score": 84,
+          "is_confirmed": true,
+          "is_machine_created": true
+        }
+      ]
+    }
+  ]
+}
+```
+  - response: same as GET for individual image above, with modified annotations
```

Markdownlint

Expected: 0; Actual: 2
Unordered list indentation

(MD007, ul-indent)